examples/async/src/components/Picker.jsx

```diff
@@ -1,6 +1,7 @@
-import React, { Component, PropTypes } from 'react'
+import React from 'react'
```

Comment:
correct (typing wise, and probably more future-proof) is `import * as React from 'react'`, both versions work now, because react is packages solely as CJS package, but this might change in the future